deployment/legacy-migration/maintenance-worker/migration/mongo/import.sh

```diff
@@ -9,16 +9,17 @@ echo "MONGODB_DATABASE         ${MONGODB_DATABASE}"
 echo "MONGODB_AUTH_DB          ${MONGODB_AUTH_DB}"
 echo "-------------------------------------------------"
 
-[ -z "$SSH_PRIVATE_KEY" ] || create_private_ssh_key_from_env
 
 rm -rf /tmp/mongo-export/*
-mkdir -p /tmp/mongo-export
+mkdir -p /tmp/mongo-export/
 
 ssh -4 -M -S my-ctrl-socket -fnNT -L 27018:localhost:27017 -l ${SSH_USERNAME} ${SSH_HOST}
 
 for collection in "categories" "badges" "users" "contributions" "comments" "follows" "shouts"
 do
-  mongoexport --host localhost -d ${MONGODB_DATABASE} --port 27018 --username ${MONGODB_USERNAME} --password ${MONGODB_PASSWORD} --authenticationDatabase ${MONGODB_AUTH_DB} --db ${MONGODB_DATABASE} --collection $collection --out "/tmp/mongo-export/$collection.json"
+  mongoexport --db ${MONGODB_DATABASE} --host localhost -d ${MONGODB_DATABASE} --port 27018 --username ${MONGODB_USERNAME} --password ${MONGODB_PASSWORD} --authenticationDatabase ${MONGODB_AUTH_DB} --collection $collection  --collection $collection --out "/tmp/mongo-export/$collection.json"
+  mkdir -p /tmp/mongo-export/splits/$collection/
+  split -l 500 /tmp/mongo-export/$collection.json /tmp/mongo-export/splits/$collection/
```

Comment:
Slit is limited to max 676 parts as it used two letters, you should append -a 3 which used three letters. Also go with maybe 1000 lines?

Reply:
Done!